seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-fake/src/main/java/org/apache/seatunnel/flink/fake/source/FakeSource.java

```diff
@@ -62,13 +111,25 @@ public String getPluginName() {
 
     @Override
     public DataSet<Row> getData(FlinkEnvironment env) {
-        Random random = new Random();
-        return env.getBatchTableEnvironment().toDataSet(
-            env.getBatchTableEnvironment().fromValues(
-                DataTypes.ROW(DataTypes.FIELD("name", DataTypes.STRING()),
-                    DataTypes.FIELD("age", DataTypes.INT())),
-                Arrays.stream(NAME_ARRAY).map(n -> Row.of(n, random.nextInt(AGE_LIMIT)))
-                    .collect(Collectors.toList())), Row.class);
+        if (this.mockDataEnable) {
+            List<Row> dataSet = new ArrayList<>(0);
+            for (long index = 0; index < mockDataSize; index++) {
+                dataSet.add(MockSchema.mockRowData(mockDataSchema));
+            }
+            return env.getBatchEnvironment()
+                .fromCollection(
+                    dataSet,
+                    MockSchema.mockRowTypeInfo(mockDataSchema)
+                );
+        } else {
+            Random random = new Random();
+            return env.getBatchTableEnvironment().toDataSet(
+                env.getBatchTableEnvironment().fromValues(
+                    DataTypes.ROW(DataTypes.FIELD("name", DataTypes.STRING()),
+                        DataTypes.FIELD("age", DataTypes.INT())),
+                    Arrays.stream(NAME_ARRAY).map(n -> Row.of(n, random.nextInt(AGE_LIMIT)))
+                        .collect(Collectors.toList())), Row.class);
```

Comment:
Can we add a default data scheme to cover this case?

Reply:
It good idea.  Have a moment.